kube-runtime/src/reflector/object_ref.rs

```diff
@@ -201,6 +202,13 @@ impl<K: Lookup> ObjectRef<K> {
         obj.to_object_ref(dyntype)
     }
 
+    pub fn from_shared_obj_with(obj: &Arc<K>, dyntype: K::DynamicType) -> Self
+    where
+        K: Lookup,
+    {
+        obj.as_ref().to_object_ref(dyntype)
+    }
+
```

Comment:
is this function necessary? can we not call `from_obj_with(&*obj, dynype)` internally?

Reply:
nope! nice, good spot, I actually moved away from using it a while back. This function is unused; binned it.